charts/incubator/invidious/values.yaml

```diff
@@ -3,13 +3,72 @@ image:
   pullPolicy: IfNotPresent
   tag: latest@sha256:3bb9e44e7d69baf91728c0a88115ef5fdb3adc91432b851e11e4d0e78bbe6e7a
 
-podSecurityContext:
-  runAsGroup: 0
-  runAsUser: 0
-
 securityContext:
-  readOnlyRootFilesystem: false
-  runAsNonRoot: false
+  container:
+    readOnlyRootFilesystem: false
+    runAsNonRoot: false
+    runAsGroup: 0
+    runAsUser: 0
+
+service:
+  main:
+    ports:
+      main:
+        port: 10594
+        protocol: http
+
+workload:
+  main:
+    podSpec:
+      containers:
+        initContainers:
+          dbseed:
+            type: install
+            enabled: true
+            env:
+              POSTGRES_DB:
+                secretKeyRef:
+                  name: cnpg-main-urls
+                  key: std
+              POSTGRES_USER: "{{ .Values.cnpg.main.user }}"
+              PGPORT: "5432"
+              PGPASSWORD:
+                secretKeyRef:
+                  name: cnpg-main-user
+                  key: password
+            command:
+              - "/bin/sh"
+            args:
+              - -c
+              - |
+                echo "Starting DB Seed..."
+                mkdir -p invidious && cd invidious
+
+                echo "Fetching seed files..."
+                git init && \
+                git remote add invidious https://github.com/iv-org/invidious.git && \
+                git fetch invidious && \
+                # Fetch config and docker dirs
+                git checkout invidious/master -- docker config
+
+                # Move config into docker dir
+                echo "Preparing directory structure..."
+                mv -fv config docker
+
+                echo "Performing the seed..."
+                cd docker
+                ./init-invidious-db.sh
```

Comment:
Not sure about this bit and how to render it correctly. The CI installs it just fine but won't run this init container which I gather is still required.